### DISCUSSION TITLE
Práctica 02: Pastelería - Terminada

### DISCUSSION THREAD

**Discussion by BarbaraMartel** [Launch-X-Latam/MisionFrontEnd#266] [Show and tell]
Buenas tardes a todos. Les presento mi práctica 02, respecto al caso de la página web de una pastelería. Utilicé un poco de CSS para darle una mayor estructura a la web con los atributos flex. Tomé algunas capturas de las páginas del index y la de pedidos, para que puedan apreciarlo de manera sencilla.

Aquí mi repositorio: [Práctica 02 - Pastelería](https://github.com/BarbaraMartel/PracticasMisionFrontEnd/tree/main/Practica-02)

**Comment by Cerecero** [comment]
Se ve muy genial, tambien puedes usar [Github Pages](https://pages.github.com/) para montar tu pagina en linea en lugar de usar imagenes puedas mostrar la pagina web.

**Reply by BarbaraMartel** [reply]
No sabía que eso se podía. Gracias por el consejo, voy a intentarlo.

**Reply by eskaider** [reply]
tambien puedes usar netlify para poder subir tu pagina a internet solo subiendo tu repositorio
